Cargo.toml

```diff
@@ -6,3 +6,7 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+tokio = { version = "1", features = ["full"] }
+anyhow = "1.0"
+futures = "0.3"
+clap = "2.33"
```

Comment:
Some of these libs are outdated and we should eval whether or not we want to use these

Reply:
Okay I will check on this soon